packages/@react-spectrum/card/test/CardView.test.js

```diff
@@ -1199,7 +1199,12 @@ describe('CardView', function () {
       let grid = tree.getByRole('grid');
       grid.scrollTop = 3000;
       fireEvent.scroll(grid);
-      expect(onLoadMore).toHaveBeenCalledTimes(1);
+      let isReact19 = parseInt(React.version, 10) >= 19;
+      if (isReact19 && layout !== GridLayout) {
+        expect(onLoadMore).toHaveBeenCalledTimes(2);
```

Comment:
?

Reply:
these loadmore checks have always been sensitive to any change, I've checked it in the browser and didn't notice an issue
I'll put in a todo to address it, though almost anything with an isOldReact or isReact19 check should be updated/looked into more at some point